### ISSUE TITLE
View/edit source is not global focus tracker friendly

### ISSUE THREAD

**Issue by hatpick** [ckeditor/ckeditor5#9906]
## 📝 Provide detailed reproduction steps (if any)

1. Add view source editing to toolbar
2. Switch to editing source
3. editor.ui.focusTracker change:isFocused is called with isFocused false.

### ✔️ Expected result

editor.ui.focusTracker is a global focus tracker, as far as its concern, editor is still focused when it's in source edit mode.

Also, focus and blur of the editor UI in this mode should also emit change:isFocused on focusTracker

### ❌ Actual result

focusTracker change:isFocused should not be called. and isFocused should be true.

## 📃 Other details

* Browser: Edge Chromium Version 91.0.864.41 (Official build) (x86_64)
* OS: macOS Big Sur 11.4
* First affected CKEditor version: 28.0.0
* Installed CKEditor plugins: @ckeditor/ckeditor5-source-editing

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by hatpick** [comment]
same goes for registerer ```editor.keystrokes.set``` observers, edit text plugin should broadcast those as well.

**Comment by Reinmar** [comment]
Thanks – nice catches!
